### PR TITLE
Define "RUNNER" environment variable based on the config file.

### PR DESCRIPTION
**Pull request by sknight80**
Define "RUNNER" environment variable during the initialization phase, so the test cases are able to use this information.

### CODE REVIEW

File: lib/chemistrykit/cli/cli.rb [hunk 1]
```diff
@@ -224,6 +224,8 @@ def rspec_config(config)
           c.before(:all) do
             @config         = config
             ENV['BASE_URL'] = @config.base_url # assign base url to env variable for formulas
+            selenium_connect = config.selenium_connect.dup
```
Comment (sknight80) [inline lib/chemistrykit/cli/cli.rb]: what does ".dup" do at the end?

Comment (aliking) [reply lib/chemistrykit/cli/cli.rb]: It returns a 'shallow' copy of the object. It's sometimes helpful when dealing with Ruby objects to make sure that you're dealing with a copy of an object and not a new object that has intact references to things inside the object. That can lead to weird behavior where you change something in what you think is a copy, and it updates a referenced object in the thing you copied from. Terrible side effects. 

Here, it doesn't look like you need it. Could you just do?:
 `ENV['RUNNER'] = config.selenium_connect[:host] || ''` ( the || allows for the value to be nil)
or even:
`ENV['RUNNER'] = config.selenium_connect[:host]` ( does it matter if this value is nil?)

Comment (sknight80) [reply lib/chemistrykit/cli/cli.rb]: I used the first version since I am checking the value of ENV['RUNNER'].

**Comment by sknight80** [comment]
:) One line change. :D

**Pull request merged**